src/header/header.scss

```diff
@@ -34,3 +34,10 @@
     text-decoration: none;
   }
 }
+
+.Header-input {
+  color: var(--color-header-text);
+  background-color: var(--color-header-search-bg);
+  border: $border-width $border-style var(--color-header-search-border);
```

Comment:
@simurai These were the variables used in dotcom, but I think they're slated to be deprecated. Should we add to https://github.com/primer/primitives/blob/d607f36c4c974017d6974382f9bb88d6418211b8/data/colors_v2/vars/component_dark.ts#L23-L26 a `border` and `inset` variable? We could also have something for buttons maybe?

Reply:
Yeah, there used to be the [globalNav](https://github.com/primer/primitives/blob/785a6d325d2a3fc3849e9e0353ce48b9229221ba/data/colors_v2/vars/deprecated.ts#L579-L588) colors, but not sure where they were used.

I think this PR is good as is, but I guess we should move the `header-search` variables that are currently in "[product](https://github.com/primer/primitives/blob/785a6d325d2a3fc3849e9e0353ce48b9229221ba/data/colors_v2/vars/product_light.ts#L164-L167)" to "[component](https://github.com/primer/primitives/blob/d607f36c4c974017d6974382f9bb88d6418211b8/data/colors_v2/vars/component_dark.ts#L23-L26)". Now that the component supports the input.

Reply:
**Update**: Moved in https://github.com/primer/primitives/pull/181